apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-settings/facility-settings-owners/facility-settings-owners.component.html

```diff
@@ -1 +1,15 @@
-<p>facility-settings-owners works!</p>
+<h1 class="page-subtitle">{{'FACILITY_DETAIL.SETTINGS.OWNERS.TITLE' | translate}}</h1>
+<perun-web-apps-refresh-button (refresh)="refreshTable()" *ngIf="!loading"></perun-web-apps-refresh-button>
+<button (click)="onCreate()" class="mr-2" color="accent" mat-flat-button>
+  {{'FACILITY_DETAIL.SETTINGS.OWNERS.ADD' | translate}}
+</button>
+<button (click)="onRemove()" [disabled]="selection.selected.length === 0" class="mr-2" color="warn" mat-flat-button>
+  {{'FACILITY_DETAIL.SETTINGS.OWNERS.DELETE' | translate}}
+</button>
+<perun-web-apps-immediate-filter (filter)="applyFilter($event)" *ngIf="!loading" [placeholder]="'VO_MANAGEMENT.FILTER_PLACEHOLDER'"></perun-web-apps-immediate-filter>
+<mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+<div *ngIf="!loading">
+  <app-alert *ngIf="!owners.length" color="warn">{{'FACILITY_DETAIL.SETTINGS.OWNERS.NO_OWNERS' | translate}}</app-alert>
```

Comment:
Please move this check into the owners-list component, so everywhere that list is used it doesnt have to be rewritten